installation/kubernetes/helm/vmclarity/values.yaml

```diff
@@ -529,7 +529,7 @@ exploitDBServer:
 
   containerSecurityContext:
     # -- Container security context enabled
-    enabled: true
+    enabled: false
```

Comment:
whats the reason for this change?

Reply:
https://openclarity.io/docs/vmclarity/getting-started/deploy-kubernetes/ if we do not disable it, the exploitDB server deployment will fail

Reply:
thank you so much for reviewing, @ramizpolic !

Reply:
lets just capture the logs on why this fails so that we can address it later. id expect that this works when the security context is enabled